HLTriggerOffline/Exotica/interface/HLTExoticaValidator.h

```diff
@@ -64,6 +64,10 @@ class HLTExoticaValidator : public DQMEDAnalyzer {
 
     /// Centralized point of access to all collections used
     EVTColContainer * _collections;
+
+    /// counting # of HLT passed events for debugging
+    std::map<std::string,int> _triggerCounter;
```

Comment:
I'm afraid this will not work the way you seem to intend. This is a DQMAnalyzer which means it is a stream type module. Therefore for each stream (which at the moment we have one stream per CPU core used in the job) there will be a different copy of this module. Each stream only sees a fraction of the total number of events and therefore each instance of this module will only see a fraction of the events. This means the count you are keeping will be incomplete since if there are 4 streams in a job each instance of the module will only see about 1/4th of the total number of events processed in the job.